fuel-core-interfaces/src/db.rs

```diff
@@ -577,4 +606,159 @@ pub mod helpers {
             Ok(self.contract.get(&contract_id).is_some())
         }
     }
+
+    /* RELAYER needs
+    Storage<Bytes32, DepositCoin, Error = KvStoreError> // token deposit
+    Storage<Address, u64,Error = KvStoreError> // validator set
+    Storage<u64, HashMap<Address, u64>,Error = KvStoreError> // validator set diff
+    */
```

Comment:
Is this a todo, or just a note?

Reply:
Just a note, but it does look ugly. Will remove it.